src/views/preview/stats.jsx

```diff
@@ -16,20 +15,20 @@ const Stats = props => (
             key="loves"
             onClick={props.onLoveClicked}
         >
-            {approx(Math.max(0, props.loveCount), {decimal: false})}
+            {Math.max(0, props.loveCount)}
```

Comment:
We should probably localize the numbers with `FormattedNumber` from `react-intl`: https://github.com/yahoo/react-intl/wiki/Components#number-formatting-components
```
<FormattedNumber value={Math.max(0, props.loveCount)} />
```

Reply:
Do we actually want commas and stuff? I thought we said we just wanted to go back to just numbers

Reply:
Some non-latin languages actually use different numbers, but to avoid risk let's just file a followup issue to really localize the numbers.